scripts/promote-cherry-picks.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Promotes cherry-picked release in related channels.
+ */
+
+import yargs from 'yargs/yargs';
+import {Prefixes, Versions} from '../configs/schemas/versions';
+import {createVersionsUpdatePullRequest, runPromoteJob} from './promote-job';
+
+interface Args {
+  amp_version: string;
+}
+const {amp_version: ampVersion}: Args = yargs(process.argv.slice(2))
+  .options({
+    amp_version: {type: 'string', demandOption: true},
+  })
+  .parseSync();
+
+const jobName = 'promote-cherry-pick.ts';
+
+function getOldAmpVersion(
+  ampVersion: string,
+  currentVersions: Versions
+): string {
+  const firstTen = ampVersion.slice(0, 10);
+  const oldVersion = Object.values(currentVersions).find(
+    (version) => version?.slice(2, 12) == firstTen
```

Comment:
Edge case we can protect against, ok to do in followup PR or just ignore: if `ampVersion` is `1234567890001`, and `version` is `1234567890002` (i.e., more advanced already) this should just throw an error